### ISSUE TITLE
Audit NumFOCUS Infrastructure

### ISSUE THREAD

**Issue by aterrel** [numfocus/infrastructure#47]
Goal: Have a written record of all servers and services, who is responsible for them, and what their purpose is

Tasks:
- [ ] Audit all the digital assets - AWS, Rackspace - turn off things we are not using.

@aterrel / @MridulS please add old audit docs here.

**Comment by adaviesNumfocus** [comment]
[Infrastructure Audit Spreadsheet](https://docs.google.com/spreadsheets/d/1BfRHVFB6H3GmwF21jvrNJ1PtAGJMdTZ79Y_kJ-R1JkU/edit?gid=0#gid=0)

**Comment by aterrel** [comment]
Perhap the next issue step is to work on #48 based on the learnings in this audit. 

Also we should document the audit in the docs in the repo, @nabobalis perhaps we can have some sort of blog post or something talking about the committee work and announcing you as the new VP of the committee?

**Comment by nabobalis** [comment]
> Also we should document the audit in the docs in the repo, @nabobalis perhaps we can have some sort of blog post or something talking about the committee work and announcing you as the new VP of the committee?

Yeah, I think we should make sure we have a whos on the committee page that I have added to the todo list.

**Comment by aterrel** [comment]
@nabobalis should we make a ticket?
